src/v2/Apps/Order/Components/__tests__/ShippingQuotes.jest.tsx

```diff
@@ -112,7 +112,7 @@ describe("ShippingQuotes", () => {
     const ascendingPrices = ["$1.00", "$2.00", "$3.00", "$4.00", "$5.00"]
 
     shippingQuotes.forEach((node: ReactWrapper, index: number) => {
-      expect(node.find("Text").first().text()).toContain(ascendingPrices[index])
```

Comment:
non-blocking because this is a test, but can we update this to target the element in a more predictable way? `.at(1)` is prone to breaking on UI changes; using a `data-test-id` or searching for the text regardless of position would be more robust.